nc_time_axis/tests/integration/test_plot.py

```diff
@@ -1,43 +1,46 @@
 """Integration test for plotting data with non-gregorian calendar."""
 
-from __future__ import (absolute_import, division, print_function)
-from six.moves import (filter, input, map, range, zip)  # noqa
-
 import unittest
 
 import matplotlib
-matplotlib.use('agg')
-import matplotlib.pyplot as plt  # nopep8
-import cftime  # nopep8
-import numpy as np  # nopep8
 
-import nc_time_axis  # nopep8
+matplotlib.use("agg")
```

Comment:
Which versions of matplotlib are supported/ tested? This is no longer necessary in newer version (certainly > 3.1 but maybe even earlier). Might be https://matplotlib.org/stable/users/prev_whats_new/whats_new_3.0.html#improved-default-backend-selection So this could potentially be removed.

Reply:
We had some problems running Iris tests locally on desktop last year because they were tripping over some problem in TkAgg (see https://github.com/SciTools/iris/pull/3846).  So removing this line _could_ cause problems.

It's nice to have an explanation of why it was only a problem on desktop and not in the CI though, so thanks for that link @mathause!

Reply:
How about we leave this line as-is, and I'll make an issue to investigate for potential removal in a follow-up PR.

Does that sound like a reasonable plan?

Reply:
Cool, see #67 - and I've tagged it for `nc-time-axis` 1.4.0